wsd/AdminModel.cpp

```diff
@@ -1015,10 +1028,10 @@ class AggregateStats final
 
     void Print(std::ostream& oss, const std::string_view prefix, const std::string_view unit) const
     {
-        print(oss, prefix, "total", unit) << _total << '\n';
-        print(oss, prefix, "average", unit) << getIntAverage() << '\n';
-        print(oss, prefix, "min", unit) << getMin() << '\n';
-        print(oss, prefix, "max", unit) << getMax() << '\n';
+        print(oss, prefix, "total", unit) << ' ' << _total << std::endl;
```

Comment:
I see the regression, although I'm not sure how I didn't notice that a space was missing). Thank for the fix.

`std::endl` is not needed here, because all it does is:
```
oss << '\n';
oss.flush();
```

And since flushing is unnecessary/unwanted, it's far clearer/preferred to use `'\n'`.